### ISSUE TITLE
Quick pick: Do not wrap to the other side of the list when holding up/down or page up/down

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#93921]
https://github.com/microsoft/vscode/issues/93641

Version: 1.44.0-insider (user setup)
Commit: d7d1147d6b764a48b789000c03088ee9afabcd8a
Date: 2020-03-31T08:23:56.446Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18362

---

I think the ideal behavior here is to only wrap around the list if you press up/pgup _again_ when you are on the first item of the list (and the same for the bottom). Without a guard like this is makes it hard to go back to the start of the list or to jump to the end of the list.

Basically, holding up and page down should take you to the top of the list as that's useful, wrapping around endlessly is not as it goes too fast to stop at the place you want.

Example of holding down page up

![ffcd7f25-fb8b-4c59-bc5d-2b85f779ecaf](https://user-images.githubusercontent.com/2193314/78028288-a0594800-7313-11ea-8619-35c7af75546f.gif)

**Comment by bpasero** [comment]
Have to check, this maybe a list thing.

**Comment by chrmarti** [comment]
It's QuickInput. If we agree this is undesired, I suggest to remove it (instead of refining it in a way that is hard to discover). You can then still wrap around using up/down arrows (which is more intuitive, I think).

**Comment by bpasero** [comment]
@chrmarti I agree about removing this, I also find it not very intuitive and would expect that you simply end up on the first or last item, but never wrap. Can you make this change?
